spec/controllers/projects/results_controller_spec.rb

```diff
@@ -48,13 +49,16 @@
 
       it 'only compares with the provided tag' do
         old_result = create :result,
+          project: project,
           tag: 'pull-request-1',
           example_name: 'user change user version 7 uses the correct timestamp'
         create :query, result: old_result, statement: 'select * from users'
 
         old_result = create :result,
-          tag: 'pull-request-2',
+          project: project,
+          tag: 'ramspull-request-2',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.